enigma-swing/src/main/java/cuchaz/enigma/gui/Gui.java

```diff
@@ -344,31 +394,44 @@ public void startRename(EditorPanel editor) {
 	}
 
 	public void showStructure(EditorPanel editor) {
```

Comment:
As I mentioned in `EditorTabbedPane.java`, either this method should be renamed to `updateStructure` or a new method `updateStructure` should be added (which shouldn't change the selected right panel)